src/i18n.ts

```diff
@@ -17,28 +17,26 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import i18next from 'i18next'
-import resourcesToBackend from 'i18next-resources-to-backend'
 import { initReactI18next } from 'react-i18next'
 
-i18next
-  .use(
-    resourcesToBackend((language, namespace, callback) => {
-      import(`../locales/${language}/${namespace}.json`)
-        .then((resources) => {
```

Comment:
For some reason that I did not manage to identify, `resources` only contains the single-word keys, so all translation keys with more than 1 word are simply not there...